README.md

```diff
@@ -57,25 +57,47 @@ for i in range(len(df_page)):
 | Layout       |                                        |              |                     |
 |              | Force Atlas 2                          | Single-GPU   |                     |
 | Link Analysis|                                        |              |                     |
-|              | Pagerank                               | Multiple-GPU |                     |
-|              | Personal Pagerank                      | Single-GPU  |                     |
+|              | Pagerank                               | Multi-GPU    |                     |
+|              | Personal Pagerank                      | Multi-GPU    |                     |
 |              | HITS                      				| Single-GPU   | leverages Gunrock   |
 | Link Prediction |                                     |              |                     |
 |              | Jaccard Similarity                     | Single-GPU   |                     |
 |              | Weighted Jaccard Similarity            | Single-GPU   |                     |
 |              | Overlap Similarity                     | Single-GPU   |                     |
 | Traversal    |                                        |              |                     |
-|              | Breadth First Search (BFS)             | Multiple-GPU |                     |
-|              | Single Source Shortest Path (SSSP)     | Multiple-GPU |                     |
+|              | Breadth First Search (BFS)             | Multi-GPU    |                     |
+|              | Single Source Shortest Path (SSSP)     | Multi-GPU    |                     |
 | Structure    |                                        |              |                     |
-|              | Renumbering                            | Single-GPU   | Also for multiple columns  |
-|              | Symmetrize                             | Single-GPU   |                     |
-
+|              | Renumbering                            | Single-GPU   | multiple columns, any data type  |
+|              | Symmetrize                             | Multi-GPU    |                     |
+| Other        |                                        |              |                     |

```

Comment:
Should spanning-tree algos in a `Tree` category instead of `Others`?

Reply:
once we have more Tree algos, then yes